compiler/rustc_codegen_gcc/src/lib.rs

```diff
@@ -26,12 +26,6 @@
 #![deny(clippy::pattern_type_mismatch)]
 #![allow(clippy::needless_lifetimes, clippy::uninlined_format_args)]
 
-// These crates are pulled from the sysroot because they are part of
-// rustc's public API, so we need to ensure version compatibility.
```

Comment:
Is this version compatibility still a concern?
I'm not sure what it's referring to, but I just want to make sure that removing this is not going to break something else.

Reply:
I added this comment in a previous PR as essentially a TODO for this PR. Now that crates are re-exported there's no need to pull them by name from the sysroot in order to ensure the versions match.